homeassistant/components/solax/__init__.py

```diff
@@ -1 +1,26 @@
 """The solax component."""
+import logging
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+
+from ...const import Platform
```

Comment:
Don't use relative imports that reach outside of the component package.

```suggestion
from homeassistant.const import Platform
```